moduleframework/common.py

```diff
@@ -242,6 +242,10 @@ def get_if_remoterepos():
     remote_repos = os.environ.get('MTF_REMOTE_REPOS')
     return bool(remote_repos)
 
+def get_odcs_auth():
+    odcstoken = os.environ.get('MTF_ODCS')
```

Comment:
would be awesome if ODCS supported this on its own

Reply:
It would be nice. we ca file issue and in future, try to change it.